config/seata-config-core/src/main/java/io/seata/config/ConfigurationKeys.java

```diff
@@ -29,7 +29,7 @@
     /**
      * The constant FILE_ROOT_CONFIG.
      */
-    public static final String FILE_ROOT_CONFIG = "config";
+    public static final String FILE_ROOT_CONFIG = "seata";
```

Comment:
There are some places need to be changed too.

Reply:
> There are some places need to be changed too.

Has been modified.